python_agent/test_custom_env.py

```
@@ -0,0 +1,154 @@
+        self.mock_zmq_client.get_observation.return_value = obs_after_pickup
+        
+        _, reward, _, _, info = self.env.step(5) # Action 5 = pickup_bone
+        
+        self.mock_zmq_client.execute_action.assert_called_once_with(
+            "interact_ground_item", {"item_id": BONE_ITEM_ID}
+        )
+        self.assertEqual(info["action_status"]["status"], "submitted")
+        # Reward: -0.1 (cost of action) + 10.0 (pickup bonus) = 9.9
+        self.assertAlmostEqual(reward, 9.9)
+
+    def test_step_unknown_action(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_step_unknown_action`

Add return type annotation: `None`

(ANN201)